pages/api/flatfile/create-embed-space.tsx

```diff
@@ -78,14 +71,19 @@ export default async function handler(
   <h3 style="margin-top: 0px; margin-bottom: 12px;">Remember, if you need any assistance, you can always refer back to this page by clicking "Welcome" in the left-hand sidebar!</h3>
 </div>`;
 
-  const addDocumentToSpaceResponse = await addDocumentToSpace(
-    "Welcome",
-    initialDocumentBody,
-    flatfileSpaceData.id,
-    accessToken
-  );
+  const title = "Welcome";
+  const body = initialDocumentBody;
+  const spaceId = flatfileSpaceData?.id as string;
+
+  const addDocumentToSpaceResponse = await addDocumentToSpace({
+    title,
+    body,
+    spaceId,
+  });
```

Comment:
Doesn't this document get created in the listener?

Reply:
Yes. Removed